client/app/hosting/general-informations/GENERAL_INFORMATIONS.html

```diff
@@ -3,427 +3,340 @@
+                                data-on-click="setAction('cdn/order/hosting-cdn-order', hostingProxy)"></oui-action-menu-item>
+                            <oui-action-menu-item
+                                data-ng-if="hosting.hasCdn && flushCdnState === 'ok'"
+                                data-text="{{:: 'hosting_dashboard_service_flush_cdn' | translate }}"
+                                data-on-click="setAction('cdn/flush/hosting-cdn-flush')"></oui-action-menu-item>
+                            <oui-action-menu-item
+                                data-ng-if="hosting.hasCdn"
+                                data-text="{{:: 'hosting_dashboard_service_terminate_cdn' | translate }}"
+                                data-on-click="setAction('cdn/terminate/hosting-cdn-terminate')"></oui-action-menu-item>
+                        </oui-action-menu>
+                    </oui-tile-definition>
+                    <oui-tile-definition data-term="{{:: 'hosting_dashboard_service_privatesql' | translate }}">
+                        <oui-tile-description>
+                            <div data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > 0"
+                                 data-wuc-progress-bar-element-counter
+                                 data-wuc-pbec-nb-elements="hosting.sqlPriveInfo.nbDataBaseActive"
+                                 data-wuc-pbec-max-elements="hosting.sqlPriveInfo.nbDataBaseInclude"
+                                 data-wuc-pbec-upper-limit="cloudWebUnlimitedQuantity"></div>
+                            <oui-action-menu data-text="{{:: 'hosting_dashboard_service_privatesql_more' | translate }}" data-ng-if="privateDatabasesLinked.length > 0">
+                                <div data-ng-repeat="privateDb in privateDatabasesLinked track by privateDb.name">
+                                    <oui-action-menu-item data-text="{{::privateDb.name}}" data-ng-if="!privateDb.isAdmin" data-disabled></oui-action-menu-item>
+                                    <oui-action-menu-item data-ng-if="privateDb.isAdmin"
+                                        data-text="{{::privateDb.name}}"
+                                        data-href="#/configuration/private_database/{{ privateDb.name }}">
+                                    </oui-action-menu-item>
+                                </div>
+                            </oui-action-menu>
+                        </oui-tile-description>
+                        <oui-action-menu data-compact data-align="end">
+                            <oui-action-menu-item data-ng-if="hosting.sqlPriveInfo.nbDataBaseInclude > 0 && hosting.sqlPriveInfo.nbDataBaseActive < hosting.sqlPriveInfo.nbDataBaseInclude && sqlPriveState !== 'doing'"
+                                data-text="{{:: 'hosting_dashboard_service_privatesql_active' | translate }}"
+                                data-ng-disabled="sqlPriveState === 'check' || sqlPriveState === 'doing'"
```

Comment:
https://ovh-ux.github.io/ovh-ui-kit-documentation/#!/oui-angular/action-menu : 
```suggestion
                                data-disabled="sqlPriveState === 'check' || sqlPriveState === 'doing'"
```